Realm/ObjectStore/object_store.cpp

```diff
@@ -479,3 +479,17 @@ void ObjectStore::delete_data_for_object(Group *group, const StringData &object_
     }
 }
 
+bool ObjectStore::is_empty(Group *group) {
+    Schema schema = schema_from_group(group);
```

Comment:
Just looping over the tables (and only checking ones with the class prefix) would be much faster. As it is this will probably surprise people with how expensive it is.